AuxTel/AuxTel-Troubleshooting/LATISS-Troubleshooting/ATspectrograph-failed.rst

```diff
@@ -0,0 +1,150 @@
+ 
+ 
+    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-7.0.1/lib/python3.11/site-packages/lsst/ts/salobj/topics/remote_command.py",
+    line 205,
+    in next_ackcmd raise base.AckTimeoutError( lsst.ts.salobj.base.AckTimeoutError: msg='Timed out waiting for command acknowledgement',
+    ackcmd=(ackcmd private_seqNum=1137560160, ack=<SalRetCode.CMD_NOACK: -301>, error=0, result='No command acknowledgement seen')
+
+.. _ATspectrograph-failed/Procedure-Steps:
+
+Procedure Steps
+===============
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+
+.. the link below gets an "This site can’t be reached"
+
+#. Power cycle the ATSpectrograph
+    #. Check ATSpectrograph CSC is in Standby Status.
+    #. Connect to ``http://aux-pdu-spectrograph.cp.lsst.org/``
+    #. Log in with Username/PW (it is in the AuxTel 1Password vault).
+    #. Click On/Off button for **Outlet 2** only. 
+
+   .. image:: ./_static/1_power_cycle_ATSpec.png
+      :width: 700px
+
+#. Connect to AuxTel EUI desktop
+    #. **Install Microsoft Remote Desktop**
+    #. Setup to connect to AuxTel EUI desktop.  
+        #. Click '+' button on the top menu. 
+        #. Select "Add PC" from drop-down menu. 
+        #. Click the drop-down menu "Gateway" > "Add Gateway"
+
+            .. image:: ./_static/2_connect_auxtel_EUI_part1.png
+                :width: 700px
+
+        #. Put aux-brick01.cp.lsst.org on the Gateway name
+
+            .. image:: ./_static/3_connect_auxtel_EUI_part2.png
+                :width: 700px
+
+    #. **Login with Username/PW of ATMCS/ATSpectrograph/ATDome EUI access** on 1Password Vault. 
+
+    #. **Open the new tab/window of the web browser on the remote desktop.**
+
+    #. Connect to  139.229.170.44:8000/Spectrograph.html
+
+        .. image:: ./_static/4_ACE_spec_EUI_Labview.png
+            :width: 700px
+
+    #. Click "**Re-init Axes**" Button on the EUI
```

Comment:
Buttons in EUI are formatted using https://obs-ops.lsst.io/project/observatory-operations-reST-reference.html#buttons-or-text-labels-in-interactive-programs